fec.config.js

```diff
@@ -30,10 +30,7 @@ module.exports = {
     moduleName: 'widgetLayout',
     exposes: {
       './RootApp': path.resolve(__dirname, './src/AppEntry.tsx'),
-      // TODO these will likely need to be restructured to be more consumable
-      './WidgetLayout': path.resolve(__dirname, './src/Components/DnDLayout/GridLayout.tsx'),
-      './WidgetDrawer': path.resolve(__dirname, './src/Components/WidgetDrawer/WidgetDrawer.tsx'),
-      './WidgetHeader': path.resolve(__dirname, './src/Components/Header/Header.tsx'),
+      './WidgetLayout': path.resolve(__dirname, './src/Routes/Default/Default.tsx'),
```

Comment:
I chose not to update the RootApp exposed module as that is currently being used by our demo page on stage (`/preview/staging/widget-layout`). If we'd prefer this to be the RootApp I can update the routes in chrome service.